### PR TITLE
Feature/button #13

### PR DESCRIPTION
**Pull request by Toine-ddt**

### CODE REVIEW

**Comment by jcelerier** [comment]
so, what's the status with this ?
is all the "relevant code" contained inside the two following commits ? 

```
@Toine-ddt add Button but it's does't work 4c41e72
@Toine-ddt buttons work : remote 2 score cf4d45c 
```

**Comment by Toine-ddt** [comment]
> so, what's the status with this ?
> is all the "relevant code" contained inside the two following commits ?
> 
> ```
> @Toine-ddt add Button but it's does't work 4c41e72
> @Toine-ddt buttons work : remote 2 score cf4d45c 
> ```

yes ! It is all the "revelant code" for buttons

**Comment by jcelerier** [comment]
okay I removed the unneeded commits then ; there may still be some fixes to do (e.g. the button uuid)

**Comment by jcelerier** [comment]
here's my bash history: 

```
11425  cd ossia/qml-remote/

# Update my repo
11428  git checkout feature/Button
11429  git pull
11430  git checkout main
11427  git pull

# Import the first commit on my local main
11432  git cherry-pick feature/button~1

# Fix a conflict:
11436  code src/ControlSurface/ScoreControlSurface.qml 
11439  git add src/ControlSurface/ScoreControlSurface.qml
11440  git cherry-pick --continue

# Import the second commit on top of that commit, this one had no conflicts
11441  git cherry-pick feature/button~0

# Push my "main" to the Github "feature/Button" branch
11445  git push origin --force HEAD:feature/Button

# Reset my "main" to Github's main
11446  git reset --hard origin/main

```

File: src/ControlSurface/ScoreControlSurface.qml [hunk 1]
```diff
@@ -101,8 +98,13 @@ Column {
                 case Uuid.impulseButtonUUID:
                     // Impulse Button
                     scoreImpulseButtons.appendImpulseButton(controlMessage)
+                    break
 
+                case "fb27e4cb-ea7f-41e2-ad92-2354498c1b6b":
```
Comment (jcelerier) [inline src/ControlSurface/ScoreControlSurface.qml:103]: here

**Review by jcelerier — CHANGES_REQUESTED** [review]
pls

**Comment by jcelerier** [comment]
noo all the commits are back :(

**Comment by Toine-ddt** [comment]
> noo all the commits are back :(

yes... I did not understand why ?
As I use github desktop, it is different from git bash, so I don't master everything yet. I am so sorry :(

**Comment by jcelerier** [comment]
arf yes, I had a hunch that this is because of gh desktop. I'd recommend switching to the CLI it's much less opaque

**Comment by jcelerier** [comment]
I made a new branch, feature/button-2

**Pull request closed without merging**